src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/ClusterAwareHitMissMetricsHolder.java

```diff
@@ -80,10 +80,10 @@ public void reportMetrics(Log log) {
     logMessage.append("** Performance Metrics Report for '");
     logMessage.append(this.metricName);
     logMessage.append("' **\n");
-    logMessage.append("\nNumber of reports: " + this.numberOfReports);
```

Comment:
That "+" makes extra internal copies of strings. It's better to use two separate statements like this:
logMessage.append("\nNumber of reports: ");
logMessage.append(this.numberOfReports);

It's about performance.

Reply:
yeah I think that's what this change is doing right? The + is being removed in this commit